core/protocols/aws-xml-protocol/src/main/java/software/amazon/awssdk/protocols/xml/internal/unmarshall/HeaderUnmarshaller.java

```diff
@@ -61,6 +63,8 @@ private HeaderUnmarshaller() {
 
     private static class SimpleHeaderUnmarshaller<T> implements XmlUnmarshaller<T> {
 
+        private static final Logger log = LoggerFactory.getLogger(SimpleHeaderUnmarshaller.class);
```

Comment:
We should use SDK Logger https://github.com/aws/aws-sdk-java-v2/blob/master/utils/src/main/java/software/amazon/awssdk/utils/Logger.java#L26

Reply:
Done